rcl/CMakeLists.txt

```diff
@@ -12,8 +12,8 @@ find_package(rosidl_generator_c REQUIRED)
 
 include_directories(include)
 
-include(cmake/get_default_rc_logging_implementation.cmake)
-get_default_rc_logging_implementation(RC_LOGGING_IMPL)
+include(cmake/get_default_rcl_logging_implementation.cmake)
+get_default_rcl_logging_implementation(RCL_LOGGING_IMPL)
```

Comment:
I kept this as a separate commit because I thought there might be a reason for the `rc_` prefix that I didn't understand, so if you think it should still be that, then please let me know and I'll remove this second commit.